### ISSUE TITLE
[Bug] <title>Public resume does not render on mobile

### ISSUE THREAD

**Issue by blackpearl-git** [AmruthPillai/Reactive-Resume#2115]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Self-Hosted

### Current Behavior

I have this fully up and running however am getting some display issues.  On mobile (iOS and android) it won't display regardless of how many times I refresh the page. Desktop chrome it wants the page to be refreshed once before it'll display the resume. Anyone have ideas how to get it to present without issue in these environments?

### Expected Behavior

I expected it to render the page as it normally does on first loading.

### What browsers are you seeing the problem on?

Chrome, Safari, Microsoft Edge

### What template are you using?

Pikachu

### Anything else?

https://cv.pittflix.com/alanpitts87/demo

**Comment by circa10a** [comment]
This is similar to #2120
I am also seeing this issue

**Comment by derickson2402** [comment]
This is happening for me as well. Not self-hosted, on the ```rxresu.me``` domain.

Friend on a tablet saw no preview but did get a working download PDF button.

On mobile, I see just a black screen with no buttons. There is a side scroll bar, but I cannot actually scroll. If I turn sideways, I can zoom out and scroll, which lets me see the download button.

**Comment by chrisbues** [comment]
Seeing this as well with rxresu.me as well. Same behaivor as above.

In the meantime, is there a way to link directly to the PDF export?

**Comment by kernelb00t** [comment]
@chrisbues Yes, on desktop in the toolbar at the bottom, the link icon next to the PDF one.

It's kind of annoying, a lot of HR managers use their phone to check CVs on the go, being not mobile responsive for the public facing part is a big red flag for them.

**Comment by crueber** [comment]
It appears that it is rendering the resume in an iframe for some reason- Most mobile devices are not iframe friendly anymore. 

I may take a shot at fixing this when I have some time, because this is a deal breaker for using the software for anything other than generating PDF resumes for me.

**Comment by theyoungwolf-dev** [comment]
Can confirm that it is still happening. Any update @crueber if you find the time, please?

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
